app/services/analytics_events.rb

```diff
@@ -967,27 +967,54 @@ def idv_forgot_password_confirmed(proofing_components: nil, **extra)
     )
   end
 
-  # @param [DateTime] enqueued_at
-  # @param [Boolean] resend
+  # @param [DateTime] enqueued_at When letter was enqueued
+  # @param [Boolean] resend User requested a second (or more) letter
+  # @param [DateTime] first_letter_requested_at When the profile became gpo_pending
+  # @param [Integer] hours_since_first_letter Diff btw first_letter_requested_at and now in hours
```

Comment:
is the abbreviation to `btw` and `Diff` to satisfy the linter? If so, it's probably better to expand and go to a new line
```suggestion
  # @param [Integer] hours_since_first_letter Difference between first_letter_requested_at
  # and now in hours
```